ambry-commons/src/test/java/com.github.ambry.commons/BlobIdTest.java

```diff
@@ -40,6 +43,7 @@
 import static com.github.ambry.clustermap.ClusterMapUtils.*;
 import static com.github.ambry.commons.BlobId.*;
 import static com.github.ambry.utils.Utils.*;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
is this a direct dependency?

Reply:
Yes, this is part of junit and is used for the `assertThat(..., not(...))` tests.